src/SDKs/SqlManagement/Management.Sql/Generated/SdkInfo_SqlManagementClient.cs

```diff
@@ -99,16 +101,5 @@ internal static partial class SdkInfo
               }.AsEnumerable();
           }
       }
-      // BEGIN: Code Generation Metadata Section
```

Comment:
Please pull latest changes from psSdkJson6, run `git clean -xdff` followed by `msbuild build.proj` and regenerate the code

Reply:
I tried this, as well as repeating the whole process. But it didn't generate this part of the code.